app/models/user.rb

```diff
@@ -55,6 +55,20 @@ def send_activation_email
     UserMailer.account_activation(self).deliver_now
   end
 
+  def create_reset_digest
+    self.reset_token = User.new_token
+    update_attribute(:reset_digest, User.digest(reset_token))
+    update_attribute(:reset_sent_at, Time.zone.now)
```

Comment:
```suggestion
    update reset_digest: User.digest(reset_token), reset_sent_at: Time.zone.now
```

Reply:
sorry Mr Bui,the oneline code is not working,i cannot reset my password if use the oneline code